lib/config/options.rb

```diff
@@ -122,7 +122,7 @@ def merge!(hash)
     def [](param)
       return super if SETTINGS_RESERVED_NAMES.include?(param)
       return super if RAILS_RESERVED_NAMES.include?(param)
-      send("#{param}")
+      public_send("#{param}")
```

Comment:
The reason I understand that this change works is because `lambda` and `proc` are both private methods (presumably of `Kernel`). Does that imply that we can remove some things out of `SETTINGS_RESERVED_NAMES` safely (like `exit!`)?

Reply:
Yes we can
I will do this